packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -528,9 +528,9 @@ export default function (Commands, Cypress, cy, state, config) {
       // we are not limited to "within" subject
       // https://github.com/cypress-io/cypress/issues/14253
       if (userOptions.capture !== 'runner') {
-        const withinSubject = state('withinSubject')
+        const withinSubject = cy.getSubjectFromChain(cy.state('withinSubjectChain') || [])
```

Comment:
I'd be nice if getSubjectFromChain would handle checking for undefined.

Reply:
Yeah, I've been thinking the same thing.

I'm going to do it in the next PR - I already have some changes queued to the subject API, so rolling it in there makes sense.